src/Sql/Vault/dbo/Stored Procedures/SecurityTask/SecurityTask_DeleteById.sql

```diff
@@ -0,0 +1,11 @@
+CREATE PROCEDURE [dbo].[SecurityTask_DeleteById]
```

Comment:
❓ Is this even doable? I'd think we don't support it. I get that orgs or ciphers would cascade a delete, but we wouldn't directly delete a task.

Reply:
That's a great point, I had added the delete sproc to complete the base `Repository` implementation. But, you're right that we don't currently plan to support deleting a specific task at this time.

Reply:
❓ how do we plan on maintaining the `SecurityTask` table?

Reply:
Removed `SecurityTask_DeleteById` in [cef36f1](https://github.com/bitwarden/server/pull/5025/commits/cef36f1796b44c41e86f8f7ef342b62e5b7dd46b)

@gbubemismith I believe the intention is to leave Tasks around for historical reporting so they won't be removed unless the organization. Otherwise, I imagine we could create background jobs that periodically clean up old/completed tasks if that's desired eventually.